docs/_running_external.rst

```diff
@@ -0,0 +1,223 @@
+workflow and the input datasets get newly uploaded at each execution. If you
+want to run the same workflow multiple times, you may prefer to avoid this.
+In the examples given so far, all workflows and datasets are specified by means
+of a local path, but Planemo also allows you to use the IDs created by Galaxy
+as well. These IDs are unique to each Galaxy server, so this approach isn't
+transferrable if you want to run your workflows on multiple servers.
+
+The first step is to ensure all the datasets which are required for the
+workflow are already uploaded. You can either do this by running the workflow
+once in the normal way, as described above, or just manually uploading through
+the web interface.
+
+To get dataset IDs, you can click on the dataset's 'View details' button (a
+small letter 'i' in a circle). This provides various information about the
+dataset and the job which created it. Under the 'Job information' section,
+there is a row named 'History Content API ID'. For each input dataset, copy
+this string (it will probably look something like ``457d46215431cc37baf96108ad87f351``)
+and paste it into the workflow job file so it looks something like the following:
+
+::
+
+    GSE37268_mof3:
+      class: File
+      galaxy_id: "457d46215431cc37baf96108ad87f351"
+    Genes:
+      class: File
+      galaxy_id: "55f30adf41ae36455431abeaa185ed89"
+
+
+i.e. just replace the ``path`` line with ``galaxy_id``.
+
+You can do exactly the same with a collection; either of the following will
+work:
+
+::
+
+    input_collection2:
+      class: Collection
+      galaxy_id: "9d362c51f575db89"
+    input_collection2:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: todo_element_name
+        galaxy_id: "457d46215431cc37baf96108ad87f351"
+
+
+In the first example, an existing collection will be used (by specifying its
+collection ID), whereas in the second, a new collection will be created from an
+existing dataset.
+
+Once the job file has been modified, run ``planemo run`` as before. The result
+should be the same, though it should be a bit faster, since the upload step was
+skipped. Instead, the selected datasets get copied to a new history, which
+unlike a new upload, doesn't result in any additional storage being used.
+
+To run the workflow using a workflow ID, replace the workflow file path with
+the workflow ID from the Galaxy server:
+
+::
+
+    $ planemo run 501da2f0ba775fd0 wf3-shed-tools-job.yml --profile tutorial_profile
+
+
+Using aliases
+===============================================
+
+Once you are dealing with a large number of workflows and datasets, you may
+find that it becomes difficult to keep track of the file paths or IDs
+which you are using for execution, particularly if you are executing workflows
+based on their ID. Planemo offers the option to create aliases, or easily
+remembrable mnemonics, for Galaxy workflows, with the following command:
```

Comment:
```suggestion
rememberable mnemonics, for Galaxy workflows, with the following command:
```

Reply:
`memorable` I suppose would be better than either :laughing: